### PR TITLE
Measure Integration Tests

### PR DESCRIPTION
**Pull request by holmanb**
```
test: Collect integration test time by default
```

### CODE REVIEW

**Comment by holmanb** [comment]
This stems from discussion last week about increasing total runtime of CI integration tests. 
```
============================= slowest 10 durations =============================

136.23s setup    tests/integration_tests/modules/test_combined.py::TestCombined::test_final_message

134.97s setup    tests/integration_tests/test_shell_script_by_frequency.py::test_per_freq

118.48s setup    tests/integration_tests/modules/test_set_hostname.py::TestHostname::test_hostname_and_fqdn

116.99s setup    tests/integration_tests/modules/test_write_files.py::TestWriteFiles::test_write_files[file /root/file_b64-ASCII text]

116.65s setup    tests/integration_tests/modules/test_set_hostname.py::TestHostname::test_prefer_short_hostname

116.54s setup    tests/integration_tests/modules/test_set_password.py::TestPasswordUsersList::test_no_duplicate_users_in_shadow

115.92s setup    tests/integration_tests/modules/test_set_hostname.py::TestHostname::test_prefer_fqdn

115.47s setup    tests/integration_tests/modules/test_ssh_auth_key_fingerprints.py::TestSshAuthkeyFingerprints::test_ssh_authkey_fingerprints_disable

115.39s setup    tests/integration_tests/modules/test_set_hostname.py::TestHostname::test_hostname

115.37s setup    tests/integration_tests/modules/test_ssh_keys_provided.py::TestSshKeysProvided::test_ssh_provided_keys[/etc/ssh/ssh_host_dsa_key.pub-AAAAB3NzaC1kc3MAAACBAPkWy1zbchVIN7qTgM0/yyY8q4RZS8cNM4ZpeuE5UB/Nnr6OSU/nmbO8LuM]

=== 114 passed, 3 skipped, 130 deselected, 16 warnings in 1650.68s (0:27:30) ===
```

**Review by TheRealFalcon — APPROVED** [review]
Is the coverage just for your own curiosity? The goal for integration tests will never be 100%, or even close.

**Comment by holmanb** [comment]
> Is the coverage just for your own curiosity? The goal for integration tests will never be 100%, or even close.

Oops, no coverage doesn't make sense. That's just some sloppy copy-paste, I'll remove that.

File: integration-requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@
 #
 pycloudlib @ git+https://github.com/canonical/pycloudlib.git@68fe052baf6f32415b727d02ba2ba48b7a995bf2
 pytest
+pytest-cov
```
Comment (aciba90) [inline integration-requirements.txt:6]: As coverage is not needed, I think it's safe to remove this.

```suggestion
```

**Review by aciba90 — CHANGES_REQUESTED** [review]
Minor comment. Otherwise, LGTM, thanks!

**Pull request merged**